### PR TITLE
Sympy 2 amr

### PR DESCRIPTION
**Pull request by mwdchang**
### Summary
`latex_to_sympy` => `latex_to_amr`  and update logic. It is expected to consume an list of LaTeX equations in the form of a JSON-array string, eg

```
[
  "\\frac{d S(t)}{d t} = - b_q S(t) I(t)",
  "\\frac{d E(t)}{d t} = b_q S(t) I(t) - r_X E(t)",
  "\\frac{d I(t)}{d t} = r_X E(t) - g_p_q I(t)",
  "\\frac{d R(t)}{d t} = g_p_q I(t)"
]
```

and outputs, if possible, a corresponding AMR JSON-string

### Testing
With this SEIR model equation

```
[
  "\\frac{d S(t)}{d t} = - b_q S(t) I(t)",
  "\\frac{d E(t)}{d t} = b_q S(t) I(t) - r_X E(t)",
  "\\frac{d I(t)}{d t} = r_X E(t) - g_p_q I(t)",
  "\\frac{d R(t)}{d t} = g_p_q I(t)"
]
```

Goto and run `http://localhost:3000/swagger-ui/index.html#/mira-controller/latexToAMR` 
- Verify that a response is returned with a JSON string of the AMR

Check the AMR is compatible, go to `https://app.staging.terarium.ai/amr-petri-test`, and paste in the response - you need to clean up the escape double-quotes first. Terarium should be able to parse and render the model

image

### CODE REVIEW

File: packages/mira/tasks/latex_to_amr.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+from mira.modeling.amr.petrinet import template_model_to_petrinet_json
+
+def cleanup():
+    pass
+
+def main():
+    exitCode = 0
+
+    try:
+        taskrunner = TaskRunnerInterface(description="Latex to SymPy")
```
Comment (YohannParis) [inline packages/mira/tasks/latex_to_amr.py:17]: If you have to do case right for SymPy, LaTeX should also be.

```suggestion
        taskrunner = TaskRunnerInterface(description="LaTeX to SymPy")
```

File: packages/mira/tasks/latex_to_amr.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            sympy_expr = parse_latex(latex_expr)
+            sympy_exprs.append(sympy_expr)
+
+        # SymPy to MMT
+        mmt = template_model_from_sympy_odes(sympy_exprs)
+
+        # MMT to AMR
+        amr_json = template_model_to_petrinet_json(mmt)
+
+        taskrunner.log(f"Latex to AMR conversion succeeded")
```
Comment (YohannParis) [inline packages/mira/tasks/latex_to_amr.py:35]:
```suggestion
        taskrunner.log(f"LaTeX to AMR conversion succeeded")
```

**Review by YohannParis — APPROVED** [review]
Great work!

File: packages/mira/tasks/latex_to_amr.py (hunk 2 above)
Comment (mwdchang) [reply packages/mira/tasks/latex_to_amr.py]: fixed

**Review by liunelson — APPROVED** [review]

**Pull request merged**